### ISSUE TITLE
Navigation of note using arrow keys

### ISSUE THREAD

**Issue by langrock** [HappenApps/Quiver#277]
Inside the Quiver Tutorial -> Getting Started note, I cannot use the arrow down key to navigate beyond the 'Welcome to Quiver ...' cell. Starting from the bottom, I can get all the way to the note title using the up key, but I cannot go back down again. This also happens inside the 'Preferences' note; I cannot scroll past the first cell.

**Comment by ylian** [comment]
I can reproduce with the 'Preferences' note, but not the "Getting Started" note. Can you click "Help > Show Tutorial" to reset the tutorials, then check the "Getting Started" note again?

Either way it's a bug, and I will fix.

**Comment by langrock** [comment]
Indeed, resetting the tutorial fixed the issue for the first note, but not the last. Whatever is causing this for the preferences note might also have been affecting the first note. Thanks for looking into it.

Carsten
